taraxa/state/dpos/precompiled/dpos_contract.go

```diff
@@ -479,6 +479,30 @@ func (self *Contract) DistributeRewards(rewardsStats *rewards_stats.RewardsStats
 	}
 }
 
+func (self *Contract) delegate_check_amount(ctx vm.CallFrame, validator *Validator) error {
+	if self.cfg.ValidatorMaximumStake.Cmp(bigutil.Add(ctx.Value, validator.TotalStake)) == -1 {
+		return ErrValidatorsMaxStakeExceeded
+	}
+
+	if self.cfg.MinimumDeposit.Cmp(ctx.Value) == 1 {
```

Comment:
Minimum is required only if there is no delegation. If there is delegation with minimum and you want to increase is by 1 it is ok

Reply:
Are you sure that it should be this way? 
I thought we discussed some time ago that we should have a config value for minimum amount that could be delegated in transaction

Reply:
Ok, I will change it back then. Also I just realized that it is a bug. Because we agreed to skip this minimum value validation if we are creating validator from genesis

Reply:
true